[2025-05-24T22:34:44.914+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-24T22:34:45.338+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_sql_data manual__2025-05-24T21:27:11.565936+00:00 [queued]>
[2025-05-24T22:34:45.742+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: supplier_data_etl_pipeline.extract_sql_data manual__2025-05-24T21:27:11.565936+00:00 [queued]>
[2025-05-24T22:34:45.759+0000] {taskinstance.py:2303} INFO - Starting attempt 7 of 12
[2025-05-24T22:34:46.990+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): extract_sql_data> on 2025-05-24 21:27:11.565936+00:00
[2025-05-24T22:34:47.100+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1881) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-24T22:34:47.101+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'supplier_data_etl_pipeline', 'extract_sql_data', 'manual__2025-05-24T21:27:11.565936+00:00', '--job-id', '158', '--raw', '--subdir', 'DAGS_FOLDER/supplier_dag.py', '--cfg-path', '/tmp/tmpyz0ex1jq']
[2025-05-24T22:34:47.103+0000] {standard_task_runner.py:63} INFO - Started process 1900 to run task
[2025-05-24T22:34:47.103+0000] {standard_task_runner.py:91} INFO - Job 158: Subtask extract_sql_data
[2025-05-24T22:34:47.644+0000] {task_command.py:426} INFO - Running <TaskInstance: supplier_data_etl_pipeline.extract_sql_data manual__2025-05-24T21:27:11.565936+00:00 [running]> on host a7d211f7b093
[2025-05-24T22:34:48.743+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='supplier_data_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract_sql_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-24T21:27:11.565936+00:00' AIRFLOW_CTX_TRY_NUMBER='7' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-24T21:27:11.565936+00:00'
[2025-05-24T22:34:48.782+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-24T22:34:48.819+0000] {etl.py:77} INFO - connected to db source
[2025-05-24T22:34:48.947+0000] {etl.py:79} INFO - queried db source
[2025-05-24T22:34:48.979+0000] {etl.py:81} INFO - closed db connection
[2025-05-24T22:34:49.007+0000] {etl.py:83} INFO - db source pickled
[2025-05-24T22:34:49.012+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-05-24T22:34:49.013+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-24T22:34:49.557+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=supplier_data_etl_pipeline, task_id=extract_sql_data, execution_date=20250524T212711, start_date=20250524T223445, end_date=20250524T223449
[2025-05-24T22:35:18.305+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-24T22:35:19.174+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-24T22:35:19.225+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
